js/script.js

```diff
@@ -77,23 +77,19 @@ const parkingLotsStyle = {
 let locationTag = "";
 
 const urlAddress = window.location.href;
-
 if (urlAddress.indexOf("#parking-reform-map") > -1) {
-  const splitAdressArray = urlAddress.split("#");
-  if (splitAdressArray[1].indexOf("parking-reform-map") > -1) {
-    // if the tag is "parking-reform-map=....."
+  const [, splitAddressFragment] = urlAddress.split("#");
+  // If the tag is "parking-reform-map=....."
+  if (splitAddressFragment.indexOf("parking-reform-map") > -1) {
     $("#toggle-projects-by-tag").css("display", "none");
-    const locationTagUrlArray = splitAdressArray[1].split("=");
-    locationTag = locationTagUrlArray[1];
+    const [, locationTagFromUrl] = splitAddressFragment.split("=");
+    locationTag = locationTagFromUrl;
   }
 }
 
-const locationTagArray = locationTag.split("%20");
-if (locationTagArray.length > 1) {
-  locationTag = locationTagArray[0];
-  for (let i = 1; i < locationTagArray.length; i++) {
-    locationTag += ` ${  locationTagArray[i]}`;
-  }
+const [firstTag, ...remainingTags] = locationTag.split("%20");

```

Comment:
`...remainingTags` means to store all of the array except for element 0 in the variable `remainingTags`.